{"name": "test002", "status": "failed", "statusDetails": {"message": "AssertionError: assert -1 == 5", "trace": "self = <testDemo.test002.TestClass object at 0x7fbf44d4d340>\n\n    @allure.feature('allure测试')\n    @allure.story('test002')\n    def test002(self):\n        c = Calc()\n        a = c.jian(2, 3)\n>       assert a == 5\nE       assert -1 == 5\n\ntest002.py:27: AssertionError"}, "attachments": [{"name": "stdout", "source": "9fcd4b38-8b83-451f-b012-6914a701345c-attachment.txt", "type": "text/plain"}], "start": 1639302523155, "stop": 1639302523155, "uuid": "6b3911a4-457e-4ad3-8108-255929969352", "historyId": "1dbe753821056c3fd3e61cb346c54aad", "testCaseId": "750facfcfedbb613ccc33da634d5afa5", "fullName": "test002.TestClass#test002", "labels": [{"name": "feature", "value": "allure测试"}, {"name": "story", "value": "test002"}, {"name": "suite", "value": "test002"}, {"name": "subSuite", "value": "TestClass"}, {"name": "host", "value": "MBP.local"}, {"name": "thread", "value": "38789-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test002"}]}